### ISSUE TITLE
Can't use Google Drive with Opera.

### ISSUE THREAD

**Issue by biffhuxtable** [perfectapi/CKP#128]
When I try to connect CKP with Google Drive using Opera, I get "function unsupported." I'm assuming it's related to the fact that Opera doesn't connect to a Google account the way Chrome does. I realize CKP is designed for Chrome, but I figured this was worth a shot.

**Comment by DukeyToo** [comment]
Yeah, probably Opera doesn't support the `chrome.identity` API that we use to connect to Google Drive.  It is possible to do it differently, but I think it would be less usable for Chrome users.
